.travis.yml

```diff
@@ -1,43 +1,70 @@
-sudo: required
-services:
-- xvfb
+language: cpp
+os: osx
+osx_image: xcode11.2
+
+addons:
+  homebrew:
+    packages:
+      - zmq
+      - zmqpp
+      - protobuf
+      - armadillo
+      - ccache
+      - ninja
+      - qt
+
 notifications:
   email: false
+
 cache:
-- ccache
+  directories:
+    - $HOME/Library/Caches/Homebrew
+    - $HOME/.ccache
 
-language: cpp
-matrix:
-  include:
-  - name: "Xenial"
-    dist: xenial
+install:
+  - cd ~/
+  - git clone https://github.com/google/googletest.git
+  - cd googletest
+  - mkdir install
+  - cd install
+  - cmake ../  #creates a make file
+  - make #compiles Google Test
+  - sudo make install #installs Google Test
+
+script:
+  - cd ~
+  - export CC="ccache gcc"
+  - export CXX="ccache g++"
+  - export CCACHE_SLOPPINESS=pch_defines,time_macros
+  - git clone https://github.com/RoboTeamTwente/roboteam_suite roboteam_suite --recurse-submodules
+  - cd ~/roboteam_suite
+  - rm -rf roboteam_ai
```

Comment:
Why is the AI folder removed?

Reply:
The ai repo is cloned twice and we don’t need this one. In the rule below we replace roboteam-ai with the other one

Reply:
Hahaha thanks, that documentation also answered about 6 other questions I had 😆🤦‍♂